core/test/common/Testable.js

```diff
@@ -5,7 +5,7 @@ const Timer = artifacts.require("Timer");
 
 contract("Testable", function() {
   it("isTest on", async function() {
-    const testable = await TestableTest.new(Timer.address);
+    const testable = await TestableTest.new((await Timer.deployed()).address);
```

Comment:
Nit: This is inconsistent to how you got the `Timer` address in your previous file change. I get that you can no longer call `Timer.address` but then you should either make an instance of `const timer = await Timer.deployed()` then pull out `timer.address` everywhere(like you did before) or do it this way everywhere.

I think I prefer declaring a `timer` then re-using it. In you're implementation here you end up doing 2 extra `await` calls as a result of not re-using the `timer` object.